dotCMS/src/main/java/com/dotmarketing/portlets/containers/business/ContainerFactoryImpl.java

```diff
@@ -105,34 +119,60 @@ public Container find(String inode) throws DotDataException, DotSecurityExceptio
 		List<Object> paramValues =null;
 		if(params!=null && params.size()>0){
 			conditionBuffer.append(" and (");
-			paramValues = new ArrayList<Object>();
+			paramValues = new ArrayList<>();
 			int counter = 0;
 			for (Map.Entry<String, Object> entry : params.entrySet()) {
 				if(counter==0){
 					if(entry.getValue() instanceof String){
 						if(entry.getKey().equalsIgnoreCase("inode")){
-							conditionBuffer.append(" asset." + entry.getKey()+ " = '" + entry.getValue() + "'");
+							conditionBuffer.append(" asset.");
+							conditionBuffer.append(entry.getKey());
+							conditionBuffer.append(" = '");
+							conditionBuffer.append(entry.getValue());
+							conditionBuffer.append("'");
 						}else if(entry.getKey().equalsIgnoreCase("identifier")){
-							conditionBuffer.append(" asset." + entry.getKey()+ " = '" + entry.getValue() + "'");
+							conditionBuffer.append(" asset.");
+							conditionBuffer.append(entry.getKey());
+							conditionBuffer.append(" = '");
+							conditionBuffer.append(entry.getValue());
+							conditionBuffer.append("'");
 						}else{
-							conditionBuffer.append(" lower(asset." + entry.getKey()+ ") like ? ");
+							conditionBuffer.append(" lower(asset.");
+							conditionBuffer.append(entry.getKey());
+							conditionBuffer.append(") like ? ");
 							paramValues.add("%"+ ((String)entry.getValue()).toLowerCase()+"%");
 						}
 					}else{
-						conditionBuffer.append(" asset." + entry.getKey()+ " = " + entry.getValue());
+						conditionBuffer.append(" asset.");
+						conditionBuffer.append(entry.getKey());
+						conditionBuffer.append(" = ");
+						conditionBuffer.append(entry.getValue());
 					}
 				}else{
 					if(entry.getValue() instanceof String){
 						if(entry.getKey().equalsIgnoreCase("inode")){
-							conditionBuffer.append(" OR asset." + entry.getKey()+ " = '" + entry.getValue() + "'");
+							conditionBuffer.append(" OR asset.");
+							conditionBuffer.append(entry.getKey());
+							conditionBuffer.append(" = '");
+							conditionBuffer.append(entry.getValue());
+							conditionBuffer.append("'");
 						}else if(entry.getKey().equalsIgnoreCase("identifier")){

```

Comment:
![MAJOR](https://sonarsource.github.io/sonar-github/severity-major.png 'Severity: MAJOR') This branch's code block is the same as the block for the branch on line 153. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarcloud.io/coding_rules#rule_key=squid%3AS1871)